README.md

```diff
@@ -1 +1,43 @@
-# Social_media_app
\ No newline at end of file
+# socialmedia_app
+This is a social media app that allows users to create posts, like and comment on posts, and follow other users. And users can chat with eachother as well. The app uses the Django framework and the Django REST framework for the backend. The frontend is built using Vite and Vanilla JS.
+
```

Comment:
You have to mention system requirements such as python with version.

Reply:
Secondly, You miss out something here, after cloning a project we have to nativate to the project and copy .env.example file to .env file.